Int32 human_id
geometry_msgs/Point32 human_center
geometry_msgs/Point32[] keypoint_data
Bool[] keypoint_mask